src/server.ts

```diff
@@ -4,17 +4,42 @@ import { setup, serve } from 'swagger-ui-express'
 import cors from 'cors'
 import { json, urlencoded } from 'body-parser'
 import { errorHandler } from './lib/error-handler'
-
 import { RegisterRoutes } from './routes'
 import * as swaggerJson from './swagger.json'
+import { startStatusHandlers } from './lib/ServiceWatcher/serviceStatus/index'
+import { serviceState } from './lib/util/statusPoll'
 
 export default async (): Promise<Express> => {
   const app: Express = express()
-
+  const statusHandler = await startStatusHandlers()
+  const serviceStatusStrings = {
+    [serviceState.UP]: 'ok',
+    [serviceState.DOWN]: 'down',
+    [serviceState.ERROR]: 'error',
+  }
   app.use(urlencoded({ extended: true }))
   app.use(json())
   app.use(cors())
 
+  app.get('/health', async (_req, res) => {
```

Comment:
there's a health route already defined which you'll want to use instead of putting it in `server.ts`. Putting it in `/controllers` means tsoa will build the documentation for swagger https://github.com/digicatapult/dscp-matchmaker-api/blob/main/src/controllers/health/index.ts